src/list/list.model.ts

```diff
@@ -11,16 +11,32 @@ export type List = {
 	phoneNumber: string | null;
 	eventDate: string | null;
 	comment: string | null;
-	submitted: boolean;
-	verified: boolean;
+	status: ListStatus;
+	submittedAt: Date | null;
+	verified: ListVerification | null;
 	createdBy: MinimalUser;
+	ownedBy: MinimalUser;
 };
 
 export type MinimalList = Omit<
 	List,
-	'structure' | 'fields' | 'colors' | 'responsible' | 'phoneNumber' | 'comment' | 'createdBy'
+	| 'structure'
+	| 'fields'
+	| 'colors'
+	| 'responsible'
+	| 'phoneNumber'
+	| 'comment'
+	| 'createdBy'
+	| 'ownedBy'
 >;
 
+export type ListStatus = 'open' | 'submitted' | 'verified';
```

Comment:
As in backend, I think enum would look better for this type of variable.

Reply:
Discussing enums in [BE pull request](https://github.com/insektionen/cleaning-list-be/pull/24#discussion_r1117936588).

Reply:
Agreed! Solved.